code/datums/diseases/advance/symptoms/damage_converter.dm

```diff
@@ -42,6 +42,10 @@ Bonus
 
 		var/list/parts = H.get_damaged_organs(1,1) //1,1 because it needs inputs.
```

Comment:
Nope, byond doesn't require the arguments be of the right amount for user-created procs.